### PR TITLE
work in progress: flake8

### PR DESCRIPTION
**Pull request by koaning**
this is a work in progress branch. i need to see the travis status to check flake8.

### CODE REVIEW

**Comment by koaning** [comment]
this a downside of github as opposed to gitlab i think. it seems that we cannot have two parallel steps in a single stage for `flake8` vs. `unittest`. the tests currently fail but it is all in the single job. are we ok with this? 

there's other things to consider too. the `.flake8` settings file currently contains: 

```
[flake8]
max-complexity=10
max-line-length=1000
exclude = */__init__.py, */*/__init__.py
```

any opinion on these settings? i want to have a `max-line-length` but i am not sure if 79 is the best limit.

**Comment by koaning** [comment]
@rogiervandergeer this should be ready for a merge.

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-complexity=10
+max-line-length=1000
```
Comment (rogiervandergeer) [inline .flake8]: Nononononono, that ain't right.

File: test_local.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+flake8 evol
+flake8 tests
+pytest
```
Comment (rogiervandergeer) [inline test_local.sh]: Newline

File: .flake8 (hunk 1 above)
Comment (koaning) [reply .flake8]: ill set it to 120

File: test_local.sh (hunk 2 above)
Comment (koaning) [reply test_local.sh]: done.

**Comment by koaning** [comment]
@rogiervandergeer i made the changes. plz allow the merge :)

**Pull request merged**